torchci/rockset/commons/__sql/flaky_workflows_jobs.sql

```diff
@@ -16,7 +16,7 @@ with repeats as (
         j.name,
         w.name
     having
-        count(*) > 1
```

Comment:
* `:count = 0` means all failures
* `:count = 1` means all failures that have been retried at least 1